### ISSUE TITLE
Submenus should be accessible from keyboard

### ISSUE THREAD

**Issue by sdellis** [ProjectMirador/mirador#2022]
Submenus in the "Appbar" menu list should be accessible from the keyboard, but are not.

<img width="279" alt="screen shot 2019-03-01 at 4 09 32 pm" src="https://user-images.githubusercontent.com/609368/53666657-b3937300-3c3c-11e9-90de-eb98f6dffe80.png">
